filters/get.js

```diff
@@ -0,0 +1,67 @@
+const getDate = (page) => page.data.changes
+  ? page.data.changes.sort((a, b) => b.time - a.time).at(0).time
+  : page.data.draft || page.data.created || page.date;
+
+const upDated = (page) => {
+  page.date = getDate(page);
```

Comment:
Hm, it might be a bit unexpected to have a filter mutate a page object like this. Depending on when/whether this filter is called, I think the same page could have a different `date` in different contexts. It might be safer to use `getDate(page)` where you want it, instead of `page.date`? E.g.

```js
const byDate = (collection) => collection
  .sort((a, b) => getDate(b) - getDate(a));

const latest = (collection) => {
  return getDate(byDate(collection).at(0));
};
```

Reply:
ah, ok

Reply:
@jgerigmeyer the issue is that I'm not sure how to sort them without updating their data? Do you want to suggest a more complete change here?

Reply:
Refactored in https://github.com/oddbird/css-sandbox/pull/33/commits/92c0be9edec0a1be4b9f45c406a3a35efe4af88d